.github/workflows/release_and_publish.yml

```diff
@@ -0,0 +1,53 @@
+ # Release package on GitHub and publish to PyPI
+ 
+ # Important: In order to trigger this workflow for the organization
+ # repo (organzation-name/repo-name vs. user-name/repo-name), a tagged
+ # commit must be made to *organzation-name/repo-name*. If the tagged
+ # commit is made to *user-name/repo-name*, a release will be published
+ # under the user's name, not the organzation.
+ 
+ #--------------------------------------------------
+ name: Release & Publish
+ 
+ on:
+   push:
+     # Sequence of patterns matched against refs/tags
+     tags:
+       - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
+ 
+ jobs:
+   build:
+     name: Create release & publish to PyPI
+     runs-on: ubuntu-latest
+     steps:
+       - name: Checkout repo
+         uses: actions/checkout@v2
+ 
+       - name: Set up python
+         uses: actions/setup-python@v2
+         with:
+           python-version: "3.x"
+ 
+       - name: Install Dependencies
+         run: |
+           python -m pip install --upgrade pip
+           pip install setuptools wheel twine jupyter urllib3 pandas pyyaml
+           python setup.py sdist bdist_wheel
+ 
+       - name: Get the tag name
```

Comment:
These two steps ('Get the tag name' and 'Release') do work. However, in [the latest `spaghetti` release](https://github.com/pysal/spaghetti/releases/tag/v1.6.5) (and other submodules) we have condensed down to a single step using [`actions/github-script@v5`](https://github.com/pysal/spaghetti/blob/1fc9647beed70a470a5d0617436d63ecb6d79020/.github/workflows/release_and_publish.yml#L40-L48). This generates customizable Release Notes in a [`.github/release.yml`](https://github.com/pysal/spaghetti/blob/main/.github/release.yml) and the actual release. I would recommend we go with that solution, but will defer to your judgment.

You can see more examples of the generated Release Notes in [`jGaboardi/tigernet`](https://github.com/jGaboardi/tigernet/releases) v2.{7,8,9}.